docs/source/deployment/amazon_emr_serverless.md

```diff
@@ -0,0 +1,234 @@
+[pre-initialized capacity](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/pre-init-capacity.html))
+ready to run a job, and the nodes may have already used the previous version of the ECR image.
+```
+
+See details on [how to run a Spark job on EMR Serverless](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/jobs-spark.html).
+
+The following parameters need to be provided:
+- "entryPoint" - the path to S3 where your `entrypoint.py` entrypoint script is uploaded.
+- "entryPointArguments" - a list of arguments for running Kedro
+- "sparkSubmitParameters" - set your properties to use the custom Python version
+
+```{note}
+The use of an "entrypoint script" is needed because [EMR Serverless disregards `[CMD]` or `[ENTRYPOINT]` instructions
+in the Dockerfile](https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/application-custom-image.html#considerations).
+We recommend to run Kedro programmatically through an "entrypoint script" to eliminate the need to use, for example,
+a [subprocess](https://docs.python.org/3/library/subprocess.html) to invoke `kedro run`. See [FAQ](#faq) for more details.
+```
+
+
+Example using AWS CLI:
+
+```shell
+aws emr-serverless start-job-run \
+    --application-id <application-id> \
+    --execution-role-arn <execution-role-arn> \
+    --job-driver '{
+        "sparkSubmit": {
+            "entryPoint": "<s3-path-to-entrypoint-script>",
+            "entryPointArguments": ["--env", "<emr-conf>", "--runner", "ThreadRunner", "--pipeline", "<kedro-pipeline-name>"],
+            "sparkSubmitParameters": "--conf spark.emr-serverless.driverEnv.PYSPARK_DRIVER_PYTHON=/usr/.pyenv/versions/3.9.16/bin/python --conf spark.emr-serverless.driverEnv.PYSPARK_PYTHON=/usr/.pyenv/versions/3.9.16/bin/python --conf spark.executorEnv.PYSPARK_PYTHON=/usr/.pyenv/versions/3.9.16/bin/python"
+        }
+    }'
+```
+
+Enter the respective values in the placeholders above. For example, use the ARN from the job runtime role created earlier for `<execution-role-arn>`.
+
+## FAQ
+
+### How is the approach defined here different from the approach in ["Seven steps to deploy Kedro pipelines on Amazon EMR"](https://kedro.org/blog/how-to-deploy-kedro-pipelines-on-amazon-emr) on the Kedro blog?
+
+There are similarities in steps in both approaches. The key difference is that this page explains how to provide a custom Python version
+through the custom image. The blog post provides Python dependencies in a virtual environment for EMR.
+
+The approach of providing a custom image is applies to EMR *Serverless*. On EMR it would be worth considering [using a custom AMI](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-custom-ami.html), as an alternative to using [bootstrap actions](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-bootstrap.html). This has not been explored nor tested as yet.
```

Comment:
```suggestion
The approach of providing a custom image only applies to EMR *Serverless*. On EMR it would be worth considering [using a custom AMI](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-custom-ami.html), as an alternative to using [bootstrap actions](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-bootstrap.html). This has not been explored nor tested as yet.
```